dts/bindings/usb-c/usb-c-connector.yaml

```diff
@@ -0,0 +1,214 @@
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    A USB Type-C connector node represents a physical USB Type-C connector.
+    It should be a child of a USB-C interface controller or a separate node
+    when it is attached to both MUX and USB-C interface controller.
+
+    This is based on Linux, documentation:
+      https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/connector/usb-connector.yaml
```

Comment:
Linking to a repository that has rebasing in name is questionable. Also, the link is to latest version and not to the actual version that this was based on.

Did you base it on some rc version (possibly already rebased)? If you based it on Linux kernel v5.19, then 
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/devicetree/bindings/connector/usb-connector.yaml?h=v5.19&id=3d7cb6b04c3f3115719235cc6866b10326de34cd
would be a better link

Reply:
I've updated the link. Thanks

Reply:
WHile you updated the code, the commit message still has the old link.

Reply:
Done